pkg/asset/installconfig/openstack/validation/platform.go

```diff
@@ -106,13 +106,13 @@ func validatePlatformFlavor(p *openstack.Platform, ci *CloudInfo, fldPath *field
 }
 
 func validateFloatingIPs(p *openstack.Platform, ci *CloudInfo, fldPath *field.Path) (allErrs field.ErrorList) {
```

Comment:
can we add a validation that makes sure people don't set both with different values.

Reply:
I added a check that returns an error if both values are set regardless of whether they are equal or not. I.e. "you cannot specify lbFloatingIP and apiFloatingIP in the install config together".